tests/Rules.Framework.IntegrationTests/Tests/Scenario5/BestServerTests.cs

```diff
@@ -0,0 +1,152 @@
+namespace Rules.Framework.IntegrationTests.Tests.Scenario5
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using FluentAssertions;
+    using Rules.Framework.Core;
+    using Rules.Framework.IntegrationTests.Common.Scenarios.Scenario5;
+    using Rules.Framework.Providers.InMemory;
+    using Xunit;
+
+    public class BestServerTests
```

Comment:
The scenario seems to be OK, and seems to cover the exact scenario you were tackling 👌 but you are using the in-memory provider package here 😄 it is not supposed to it in this tests project. Let me ask you, does this test make sense for the core package anyway? Or is it only to test the changes on the in-memory provider project?

Reply:
Done

Reply:
Thank you Carlos 👌